lib/ae_mdw_web/controllers/block_controller.ex

```diff
@@ -171,6 +171,80 @@ defmodule AeMdwWeb.BlockController do
     end
   end
 
+  defp json_with_sorted_mbs(conn, %{next: next, data: block_list}) do
+    resp_json =
+      Enum.join([
```

Comment:
I don't know what this does, but it doesn't look right

Reply:
same for the other occurrences that manually work with json as strings.

Reply:
maybe @sborrazas can also give some feedback here?

Reply:
It's regarding to this aspect: https://github.com/aeternity/ae_mdw/issues/223#issuecomment-908477944

We can't use regular encoder because Elixir maps and Json objects don't assure the ordering so the only way without a breaking change was with a custom encoder.

Even though it's library and client dependent, in the case of the Explorer (mdw-frontend) the fix suits to the need. 
So we have three options:
I) implement a /v2 of the endpoint with the micro_blocks in a list (sorted by time)
II) accept this change to satisfy Explorer (with the risk when it changes the library, might get a different result, i.e not sorted by time).
III) do both I & II

Reply:
D) would be a braking change.

Reply:
We discussed about this issue last week w/ @jyeshe .. I don't think we should implement such hack-ish behavior for encoding JSONs. Also, like you mentioned @thepiwo, the JSON standard does not guarantee any sort of ordering on object keys. 

If it's a matter of how we're displaying things then it's a matter of UI, which means the UI should be updated. Otherwise, if we want to provide some ordering on the API JSON responses, lists should be used instead.

Reply:
I prefer also not putting this diablito (((: So to make "v1" with explicit order, I like @thepiwo suggestion to add a new field. But as the clients gonna need to change, besides this, I will provide also a v2 with the correct fix.

Reply:
Implemented sorted list. I am splitting the v1, pagination, mb limit into a following PR to better discuss pending details.